.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,7 +1,7 @@
 name: Bug Report
 description: For users or developers to report any programming issues, such as crashes, errors, or unexpected behavior of ABACUS
 labels: [Bug Report]
-assignees: dyzheng
+assignees: [insert assignee here]
```

Comment:
I wonder if this satisfies the syntax for issue template? Maybe leaving blank will be OK?

Reply:
@caic99 This is ok. You can check it with the flag "code quality", which write assignees as [insert assignee here]